aggregator/src/main/resources/application.properties

```diff
@@ -9,12 +9,20 @@ quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
 
 quarkus.hibernate-orm.physical-naming-strategy=com.redhat.cloud.notifications.db.naming.SnakeCasePhysicalNamingStrategy
 
+# Output queue
 mp.messaging.outgoing.aggregation.connector=smallrye-kafka
 mp.messaging.outgoing.aggregation.topic=platform.notifications.aggregation
 mp.messaging.outgoing.aggregation.group.id=integrations
 mp.messaging.outgoing.aggregation.key.serializer=org.apache.kafka.common.serialization.StringSerializer
 mp.messaging.outgoing.aggregation.value.serializer=org.apache.kafka.common.serialization.StringSerializer
 
+# Output queue
+mp.messaging.outgoing.egress.connector=smallrye-kafka
+mp.messaging.outgoing.egress.topic=platform.notifications.ingress
+mp.messaging.outgoing.egress.group.id=integrations
```

Comment:
This is a common "mistake" in our configuration files but `group.id` only makes sense for `incoming` channels. There is no such thing as groups for Kafka producers.
```suggestion
```